### DISCUSSION TITLE
Lando Configuration for object cache pro plugin enable

### DISCUSSION THREAD

**Discussion by BeturaDev** [community/community#137941] [Programming Help] [answered]
### Body

I'm using Lando environment for Wordpress development. I tried to use Object Cache Pro Plugin but it shows some error related Redis connection. How can I solve this problem? Thank you.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by brianshadow** [comment, marked as answer]
Hello. I met that problem last month and solved with reinstalling redis with igbinary library.
You can add these script to your lando configuration.

services:
  appserver:
    webroot: ./web
    xdebug: true
    extras:
      - apt-get update -y
      - apt-get install -y build-essential zlib1g-dev liblz4-dev libzstd-dev
      - pecl install igbinary msgpack
      - pecl uninstall -r redis
      - printf "\n" | pecl install --configureoptions='enable-redis-lzf="yes" enable-redis-zstd="yes" enable-redis-lz4="yes" enable-redis-igbinary="yes" enable-redis-msgpack="yes"' redis

**Reply by BeturaDev** [reply]
It worked. Thank you.
